spyder/plugins/findinfiles/widgets.py

```diff
@@ -852,6 +852,7 @@ def __init__(self, name=None, plugin=None, parent=None,
         self.supported_encodings = self.get_option('supported_encodings')
         self.search_thread = None
         self.running = False
+        self.extras_toolbar = None

```

Comment:
Since the `getattr` method is being used bellow, is then this attribute necessary? or maybe instead of the `getattr` we should initialize here also the `self.more_options_action`?

Reply:
Fixed!